### ISSUE TITLE
Skipper installation shows error messages

### ISSUE THREAD

**Issue by sharbov** [Stratoscale/skipper#51]
```
sudo pip install -U 'strato-skipper==1.1.2'
[sudo] password for shay: 
Collecting strato-skipper==1.1.2
  Downloading strato-skipper-1.1.2.tar.gz
Requirement already up-to-date: PyYAML>=3.11 in /usr/lib64/python2.7/site-packages (from strato-skipper==1.1.2)
Requirement already up-to-date: click>=6.6 in /usr/lib/python2.7/site-packages (from strato-skipper==1.1.2)
Requirement already up-to-date: requests>=2.6.0 in /usr/lib/python2.7/site-packages (from strato-skipper==1.1.2)
Requirement already up-to-date: tabulate>=0.7.5 in /usr/lib/python2.7/site-packages (from strato-skipper==1.1.2)
Building wheels for collected packages: strato-skipper
  Running setup.py bdist_wheel for strato-skipper ... error
  Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-mQDhdc/strato-skipper/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /tmp/tmpitcZL5pip-wheel- --python-tag cp27:
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-mQDhdc/strato-skipper/setup.py", line 9, in <module>
      raise RuntimeError("This setup.py does not support wheels")
  RuntimeError: This setup.py does not support wheels

  ----------------------------------------
  Failed building wheel for strato-skipper
  Running setup.py clean for strato-skipper
Failed to build strato-skipper
Installing collected packages: strato-skipper
  Found existing installation: strato-skipper 1.1.2.dev4
    Uninstalling strato-skipper-1.1.2.dev4:
      Successfully uninstalled strato-skipper-1.1.2.dev4
  Running setup.py install for strato-skipper ... done
Successfully installed strato-skipper-1.1.2

```

**Comment by adir-stratoscale** [comment]
@shay-stratoscale this error message is due to the workaround introduced by https://github.com/Stratoscale/skipper/commit/18369940730d0e276c1c294f87d5c8ac00de0951

Unfortunately, `pip install` goes through wheel caching, which installs data files under site-packages. Therefore, without this workaround skipper data files (i.e. bash completion and entrypoint) are not installed correctly, when running `pip install strato-skipper`.

An alternative to the above workaround would be to install skipper via `pip --no-cache-dir install strato-skipper`, which is not intuitive to me.

@shay-stratoscale do you have other alternatives?

**Comment by shay-stratoscale** [comment]
If you're the one raising the exception, just don't, print a warning and skip it instead

either way, an end user will assume something went wrong, and I think this should be unacceptable

**Comment by adir-stratoscale** [comment]
@shay-stratoscale i'm the one who raises the exception, but the the decision whether to show it to the user is done by pip internally. 
i can add to the erro message a clarification sating it is just skipping wheel.

**Comment by eran-stratoscale** [comment]
The issue was fixed by : 7fa59f27428fd3cfb708b8e89eefef131c48ca3d.
closing

**Comment by romfreiman** [comment]
Just happened again:


 Building wheel for strato-skipper (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-PJ5Fxm/strato-skipper/setup.py'"'"'; __file__='"'"'/tmp/pip-install-PJ5Fxm/strato-skipper/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-tfRpWd
       cwd: /tmp/pip-install-PJ5Fxm/strato-skipper/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-PJ5Fxm/strato-skipper/setup.py", line 8, in <module>
      raise RuntimeError("This setup.py does not support wheels")
  RuntimeError: This setup.py does not support wheels
  ----------------------------------------
  ERROR: Failed building wheel for strato-skipper
